packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/example.html

```diff
@@ -66,6 +66,20 @@ <h2>Examples</h2>
       <span slot="help">This is a help message</span>
     </gux-form-field-color>
   </fieldset>
+
+  <fieldset>
+    <legend>Tooltip</legend>
+
+    <gux-form-field-color>
+      <input slot="input" type="color" name="e-1" value="#cc3ebe" />
+      <label slot="label">Default</label>
+      <gux-form-field-label-tooltip
```

Comment:
I am not 100% convinced we need to scope the name of this to `form-field`. I am thinking maybe something like 
`gux-icon-tooltip`

Reply:
That seems sensible to me. I can see other scenarios that might require a tooltip with icon combo.